aws/eks/helm.tf

```diff
@@ -157,6 +157,13 @@ resource "helm_release" "kubernetes_external_secrets" {
   }
 }
 
+module "configmapsecrets" {
+  source                         = "github.com/mozilla-it/terraform-modules//helm/configmapsecrets?ref=master"
```

Comment:
Can we include this by relative path instead? Otherwise that module would always follow the master branch, even if the eks module is pinned to a specific version, which seems weird.